src/components/Article.jsx

```diff
@@ -9,16 +9,39 @@ const Article = () => {
     const { id } = useParams();
     const [article, setArticle] = useState({});
     const [isLoading, setIsLoading] = useState(true);
+    const [votes, setVotes] = useState(0);
 
     useEffect(() => {
         setIsLoading(true);
         getArticle(id)
         .then((article) => {
             setArticle(article);
+            setVotes(article.votes);
             setIsLoading(false);
         })
     }, []);
     
+    const handleUpvote = () => {
+        setVotes((currVotes) => {
+            return currVotes + 1;
+        });
+        // J.D: TODO: More appropriate handle any errors? Alternatively: fire & forget (remove catch)
+        patchArticleVotes(article.article_id, 1)
+        .catch((err) => {
+            alert("Oh noes, failed to upvote!")
```

Comment:
Cute message!
I see you said you might refactor this into something else - maybe a little vanishing p tag?

Reply:
Yeah, I agree with swapping out the alert for something more appropriate; I'm just reaching for using alerts for now to get the core functionality implemented, then I'll revisit when I come to add fancier styling :)